test/strategies/explicit.jl

```diff
@@ -0,0 +1,54 @@
+good = KNNClassifier(K=2)
+bad = KNNClassifier(K=10)
+ugly = ConstantClassifier()
+
+r = [good, bad, ugly]
+
+rng=StableRNGs.StableRNG(123)
+X, y = make_blobs(rng=rng)
+
+@testset "constructor" begin
+true 
+end
+
+
+@testset_accelerated "integration" accel begin
+
+    # evaluate the three models separately:
+    resampling = Holdout(fraction_train=0.6)
+    scores = map(r) do model
+        e = evaluate(model, X, y, resampling=resampling, measure=LogLoss())
+        return e.measurement[1]
+    end
+    # scores are ordered best to worst:
```

Comment:
Given that `StableRNG` should be stable over Julia versions (give the same output), these comments could be assertions as well, I think.

Reply:
I've added `@test scores == sort(scores)`. This is what you meant, yes?